php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/PhpFile.java

```diff
@@ -29,14 +28,15 @@
 
   /**
    * @return path relative to project directory
+   * @deprecated since 2.14. Use {@link PhpFile#filename()} or {@link PhpFile#toString()} (returning a string to identify this file suitable for logs).
    */
+  @Deprecated
   Path relativePath();
```

Comment:
Can we stop using relativePath [in PHPCheckVerifier](https://github.com/SonarSource/sonar-php/blob/master/php-frontend/src/main/java/org/sonar/plugins/php/api/tests/PHPCheckVerifier.java#L75)?